packages/application-components/src/components/modal-pages/tabular-modal-page/tabular-modal-page.tsx

```diff
@@ -71,7 +71,6 @@ const TabularModalPage = (props: Props) => (
       css={css`
         background-color: ${customProperties.colorNeutral95};
         padding: ${customProperties.spacingM} ${customProperties.spacingM} 0;
-        border-bottom: 1px ${customProperties.colorNeutral} solid;
```

Comment:
Why was the border removed?

Reply:
@FilPob asked me to do so 🙂

Reply:
Sorry for that, I wasn't aware of the consequences. Reverted.